### ISSUE TITLE
Not working on Microsoft Edge browser.

### ISSUE THREAD

**Issue by canewsin** [krzysztof113/ZeronetLink#2]
site not detecting ZeroNet installation on new Microsoft edge browser.

**Comment by canewsin** [comment]
ok found the problem, I was using dark mode extension, checking background colour is not reliable because extensions like, I use or chrome dark mode flag may interfere with these.

**Comment by krzysztof113** [comment]
Yes, I even made a PR to bypass that problem but @HelloZeroNet never accepted it.
https://github.com/HelloZeroNet/ZeroHello/pull/129

**Comment by HelloZeroNet** [comment]
Here is an example for easier check:
```js
elem_local_test = createElem('<img src="http://127.0.0.1:43110/favicon.ico?random=' + Date.now() + '" style="display: none"/>');

elem_local_test.onerror = function() {
    document.getElementById("visit-links").classList.add("no-local");
};
```

https://github.com/HelloZeroNet/zeronet.io/blob/master/go.html#L68

**Comment by canewsin** [comment]
@krzysztof113 what do you think about the @HelloZeroNet Suggestion ?

**Comment by krzysztof113** [comment]
Yes, I think I'll move to it. The only problem I've seen is when you have regular webserver on custom port then favicon will be there, with false positive, but the chance for that is negligible. 
Overall why are you interested? You plan to use it somewhere in your app? There are barely any users by now.

**Comment by canewsin** [comment]
i heard you are abandoning Zeronet.link redirector, i would like to took up that project & host, as you may know me as developer of zeronet mobile and i previously showed interest in this project, you can trust me to maintain the project for long.

**Comment by krzysztof113** [comment]
> i heard you are abandoning Zeronet.link redirector, i would like to took up that project & host, as you may know me as developer of zeronet mobile and i previously showed interest in this project, you can trust me to maintain the project for long.

please reach out to me @ krzysztof113 @ o2.pl

**Comment by canewsin** [comment]
@krzysztof113 is there an another way of communication to reach you ? like telegram or even matrix rooms or something else.
